### ISSUE TITLE
False positive: `schema.core/fn` is not linted correctly

### ISSUE THREAD

**Issue by mynomoto** [borkdude/clj-kondo#492]
**version**
clj-kondo v2019.09.22-alpha

**platform**
linux

**problem**
`schema.core/fn` is not linted correctly

**repro**
```
(ns yyyy
  (:require [schema.core :as s]))

(s/fn my-identity :- s/Any
  [x :- s/Any]
  x)
```

**expected behavior**
No linter complaints.

**actual behavior**
```
yyyy.cljs:4:7: info: unresolved symbol my-identity
yyyy.cljs:5:4: info: unresolved symbol x
```

**Comment by borkdude** [comment]
Fixed with https://github.com/borkdude/clj-kondo/commit/abefb1f8363f7294cf51600df50caa17d99784ed

**Comment by mynomoto** [comment]
You are amazing! Thanks!
